draft-meynell-panrg-scion-research-questions.md

```diff
@@ -196,9 +196,34 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 
 ## ISD, AS identity
 
+The SCION protocol specifies 16 bits and 48 bits to identify the ISD and AS.
+These identification is, at this moment, necessary at the data-plane level,
+written on each one of the data packets to fully address the sender and receiver,
+as well as at the control-plane level, to identify the PCB sender and hops.
+
+While 48 bits seem sufficient to accommodate a large number of ASes,
+using 16 bits for the ISD might not be.
+Additionally, the justification of these figures is lacking at the moment.
+
+The following questions arise: (not comprehensive)
+
 * How many ASes do we expect in the SCION network model?
-* How many ISDs? What is the ontology of an ISD? Per geographic area only?
-* One AS belongs to many ISDs?
+* Can one AS belong to many ISDs?
+* Are AS numbers unique themselves? Or only unique in combination with an ISD?
+* How many ISDs do we expect?
+* What is the ontology of an ISD?
+  Note that not all combinations of ASes might be possible as ISDs:
```

Comment:
Yes, it refers to an AS being part of multiple ISDs.

Reply:
Maybe it can be rephrased a bit? I think I misunderstood the meaning of "all". I think what it means is that all (=arbitrary) combinations should definitely be possible, but not (all) at the same time. 
Does that make sense?

Actually, I also think the word "combinations" is a bit misleading. I think it doesn't really matter how many combinations are available. If ISD start being used as "types", many of them may contain an identical set of ASes. We may still run out of ISD numbers, even if the number of actually used combinations is quite low....